client/my-sites/plans/current-plan/index.jsx

```diff
@@ -193,7 +195,244 @@ class CurrentPlan extends Component {
+
+				<Card className="current-plan__trial-card">
+					<div className="current-plan__trial-card-content">
+						<Badge type="info">{ translate( '5 days left' ) }</Badge>
+						<p className="current-plan__card-title">
+							{ translate( 'You’re in a free trial store' ) }
+						</p>
+						<p className="current-plan__card-subtitle">
+							{ translate(
+								'Your free trial we’ll expire in 5 days. Pick a plan by December, 13th to keep your store running.'
+							) }
+						</p>
+						<Button primary>{ translate( 'Pick a plan' ) }</Button>
+					</div>
+					<div className="current-plan__trial-ilustration">
+						<img
+							width={ 180 }
+							alt="Pick a plan"
+							src="/calypso/images/plans/wpcom/plan-ecommerce-trial.png"
+						/>
+					</div>
+				</Card>
+
+				<h2 className="current-plan__section-title">What’s included in your free trial</h2>
+				<div className="current-plan__included-wrapper">
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
+							alt="Priority support"
+							src="/calypso/images/plans/wpcom/trial-features/priority-support.svg"
+						/>
+						<p className="current-plan__included-title">Priority support</p>
+						<p className="current-plan__included-text">
+							Need help? Reach out to us anytime, anywhere.
+						</p>
+						<Button className="current-plan__included-link">Ask a question</Button>
+					</Card>
+
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
+							alt="Premium themes"
+							src="/calypso/images/plans/wpcom/trial-features/premium-themes.svg"
+						/>
+						<p className="current-plan__included-title">Premium themes</p>
+						<p className="current-plan__included-text">
+							Explore a diverse selection of beautifully designed premium themes.
+						</p>
+						<Button className="current-plan__included-link">Browse premium themes</Button>
+					</Card>
+
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
+							alt="Advanced customization"
+							src="/calypso/images/plans/wpcom/trial-features/advanced-customization.svg"
+						/>
+						<p className="current-plan__included-title">Advanced customization</p>
+						<p className="current-plan__included-text">
+							Change your store's appearance in a few clicks!
+						</p>
+						<Button className="current-plan__included-link">Design your store</Button>
+					</Card>
+
+					<Card>
+						<img
+							className="current-plan__included-ilustration"
+							alt="Unlimited products"
+							src="/calypso/images/plans/wpcom/trial-features/unlimited-products.svg"
+						/>
+						<p className="current-plan__included-title">Unlimited products</p>
+						<p className="current-plan__included-text">
+							List as many products or services as you’d like and offer subscriptions.
+						</p>
+						<Button className="current-plan__included-link">Add a product</Button>
+					</Card>
```

Comment:
It is probably worth pushing these repeated `Card` elements into a dedicated component to simplify the code and reduce repetition that is more likely to introduce bugs.